gluten-hudi/src/test/scala/org/apache/gluten/execution/HudiSuite.scala

```diff
@@ -85,7 +85,7 @@ abstract class HudiSuite extends WholeStageTransformerSuite {
   }
 
   // FIXME: flaky leaked file systems issue
-  ignore("hudi: mor", Some("3.2")) {
+  ignoreWithSpecifiedSparkVersion("hudi: mor", Some("3.2")) {
```

Comment:
this can be changed to ignore

Reply:
Previously only ignored on spark3.2. Change to ignore to ignore on all versions?

Reply:
Currently, we only set `minSparkVersion` to 3.2 with the intention of skipping the related tests for all Spark versions. Therefore, using the ignore to `bypass` the tests across all versions is a more appropriate solution.

Reply:
Sorry, I miss this point, make sense to me, addressed.